Automattic/jetpack
# Social Review Prompt: Ensure it is still shown with Jetpack Active

We had to revert #29910 in #30101 because the review prompt state was stamping on the rest of the initial state object that's used by Social/Publicize.

## Proposed changes:

This change adds the Post Publish panel with the review prompt to the publicize-components package and merges the required state in the Social plugin, so that if both plugins are active the review prompt is still shown in Jetpack.





### Other information:


## Jetpack product discussion


1198218726984184-as-1204393860935466/f


## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:
* The testing is the same as #27917 but for both Jetpack and Social plugins
* Check that you see the review prompt with Social active, but not with just Jetpack active
* Check that the prompt can be dismissed
* Check that the link takes you to review Social.
